fe/fe-core/src/main/java/com/starrocks/connector/PartitionUtil.java

```diff
@@ -653,7 +653,7 @@ public static RangePartitionDiff getPartitionDiff(Expr partitionExpr, Column par
             if (functionCallExpr.getFnName().getFunction().equalsIgnoreCase(FunctionSet.DATE_TRUNC) ||
                     functionCallExpr.getFnName().getFunction().equalsIgnoreCase(FunctionSet.STR2DATE)) {
                 return SyncPartitionUtils.getRangePartitionDiffOfExpr(basePartitionMap,
-                        mvPartitionMap, functionCallExpr, partitionColumn.getPrimitiveType());
+                        mvPartitionMap, functionCallExpr, partitionColumn.getPrimitiveType(), null);
```

Comment:
it seems that getRangePartitionDiffOfExpr without last null parameter is the same as with null. so it is not necessary to modify this line.

Reply:
The callee needs extra parameter, i don't want to add another method, so i will keep it unchanged for now.